README.md

```diff
@@ -171,5 +171,23 @@ You can use the staging broker in your tests if necessary (if it can be avoided,
 
 As there is no real value of testing non-RPC calls (you don't even know if you call it correctly :x), you can mock those.
 
+## Releases
+To do releases you will need to have [git-flow](https://github.com/petervanderdoes/gitflow-avh) installed and use it to perform a release. For more info see a [cheatsheet](https://danielkummer.github.io/git-flow-cheatsheet/).
+### Git flow set up
+To set up git flow, after instalation you need to runt: `git flow init` at the root of the project/repo and accept all default settings (development branch: develop, release branch master, no version tag prefix)
+
+### Performing a release
```

Comment:
Oh I see now, these instructions do not take into account doing a deployment per because master is updated via git flow locally. There is usually no need to review a deployment PR because in most cases it only contains the version bump.